server/src/core/python_odoo_builder.rs

```diff
@@ -398,7 +398,7 @@ impl PythonOdooBuilder {
         let base_model_syms = session.sync_odoo.get_symbol(session.sync_odoo.config.odoo_path.as_ref().unwrap(), &base_model_tree, u32::MAX);
         let model_syms = session.sync_odoo.get_symbol(session.sync_odoo.config.odoo_path.as_ref().unwrap(), &model_tree, u32::MAX);
         let transient_syms = session.sync_odoo.get_symbol(session.sync_odoo.config.odoo_path.as_ref().unwrap(), &transient_tree, u32::MAX);
-        if base_model_syms.is_empty() || model_syms.is_empty() || transient_syms.is_empty() {
+        if base_model_syms.is_empty(){
```

Comment:
I don't understand what you are trying to achieve here...

Anyway, if this is necessary, the first check in the `if` below is redundant (we know `base_model_syms` it's not empty).